config.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+const (
+	defaultListenAddr = ":8080"
```

Comment:
This changed in README to 23222

Reply:
Yes, that was to keep the command example in the readme and am-executor example config file consistent :smiley_cat: